MessageApp/Controllers/ThemesViewController.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  ThemesViewController.swift
+//  MessageApp
+//
+//  Created by Macbook on 28.09.2022.
+//
+
+import UIKit
+
+final class ThemesViewController: UIViewController {
+    
+    private lazy var themesView = ThemesView()
```

Comment:
Вместо этой вьшки ты закинешь три кнопки как-то так:

```suggestion
    let themeManager: ThemeManager
   
    private lazy var dayThemeButtom = TheemeButton(
          theme:  themeManager.currentTheme(),
          title: "Day Theme",
          action: {
              self.themeManager.applyTheme(theme: DayTheme)
          }
    )
```

При этом `ThemeManager` пусть будет синглтоном - объект, который всегда живет в памяти и хранит состояние темы

Все экраны берут у `ThemeManager` тему, а `ThemeViewController` может эту тему у `ThemeManager` еще и поменять